rasterio/rio/rm.py

```diff
@@ -0,0 +1,33 @@
+"""``$ rio rm``"""
+
+
+import click
+
+import rasterio
+from rasterio.errors import DriverRegistrationError, RasterioIOError
+
+
+@click.command()
+@click.argument('path')
+@click.confirmation_option(
+    help="Confirm delete without prompting",
+    prompt="Are you sure you want to delete the dataset?",
+    expose_value=True)
+@click.option(
+    '-f', '--format', '--driver',
```

Comment:
Too bad about the short option conflict between `--format` and `--force`.

Reply:
@sgillies Whats the reasoning for `--force-overwrite` vs. just `--overwrite`?  `-o` is mapped to `--outfile`?  We could just move towards dropping the `-f` and use `--overwrite`.  `-f` for output format is certainly familiar coming from GDAL and OGR's `-f` and `-of`.

Reply:
@geowurster that's a good idea. I thought `--force-overwrite` might eventually add `-f` like rm, but it's better to reserve `-f` for format.

Reply:
Handling this in https://github.com/mapbox/rasterio/issues/1181.